README.md

```diff
@@ -224,6 +224,51 @@ See [using Ansible collections](https://docs.ansible.com/ansible/devel/user_guid
    > [!NOTE]
    Add this certificate to all RHTAS client nodes that use the `cosign` and `gitsign` binaries for signing and verifying artifacts.
 
+## Configuring a user-provisioned MariaDB and Redis instance for the RHTAS Ansible Collection
+    
+Prerequisites:
+    
+* An Amazon Web Services (AWS) account with the ability to create MariaDB and Elasticache Redis instances or equivalent.
+* The RHTAS Ansible Collection installed and configured.
+
+Steps:
+
+1. Create a MariaDB instance:
+    * Follow the MariaDB setup documentation for RHTAS found [here](https://docs.redhat.com/en/documentation/red_hat_trusted_artifact_signer/1/html-single/deployment_guide/index#configuring-amazon-rds-for-trusted-artifact-signer_deploy).
+    * Ensure the instance uses the Trillian schema.
+    * Note the instance's hostname, port, username, root password, and password.
+
+2. Configure the Ansible collection:
+    * Within `roles/tas_single_node/defaults/main.yml` change the following
```

Comment:
We can't tell users to change this file. Users would need to put this in their playbook (like the one that has an example in this README)